Control-User-Interfaces/EFD.rst

```diff
@@ -34,9 +40,16 @@ Chronograf
 .. note::
 
   To login to Chronograf, you need to be a member of the GitHub lsst-sqre organization. If you are not, please request access on the ``#com-square`` LSSTC Slack channel.
+  You will also need access to the appropriate VPN to log in.
+  For information on the US data facility (NCSA) VPN see `instructions here <https://wiki.ncsa.illinois.edu/display/cybersec/Virtual+Private+Network+(VPN)+Service#VirtualPrivateNetwork(VPN)Service-UsingtheCiscoAnyConnectVPNClient(Required)>`_.
+  For information about connecting to the Chilean VPN, please ask on the ``#rubinobs-it-chile`` LSSTC Slack channel.
 
 .. Would be nice to show some sort of screenshot here?
 
+.. figure:: /weather.png
```

Comment:
This figure isn't showing up in the html render. I can fix it if you'd like though.